src/script/components/RichTextEditor/plugins/TypeaheadMenuPlugin.tsx

```diff
@@ -453,7 +453,7 @@ function LexicalPopoverMenu<TOption extends TypeaheadOption>({
           selectOptionAndCleanUp(options[selectedIndex]);
           return true;
         },
-        COMMAND_PRIORITY_LOW,
```

Comment:
doesn't this conflicts with the changes to the send plugin?

Reply:
Well it would indeed conflict if it was left with the same priority, but now since it was changed to NORMAL (so higher then the send plugin one), it will still be executed.